packages/apps/hr/src/components/molecules/TimesheetDetailCard/TimesheetDetailCard.tsx

```diff
@@ -100,7 +97,22 @@ const TimesheetDetailCard = ({
   }, [item.id]);
 
   return (
-    <View style={[styles.container, style]}>
+    <ActionCard
+      translator={I18n.t}
+      actionList={[
+        {
+          iconName: 'send-fill',
+          onPress: onSend,
+          hidden:
+            !_isActions || _statusSelect !== Timesheet?.statusSelect.Draft,
+        },
+        {
+          iconName: 'check-lg',
+          onPress: onValidate,
+          hidden:
+            !_isActions || _statusSelect === Timesheet?.statusSelect.Draft,
+        },
+      ]}>
```

Comment:
Maybe we can return undefined or empty list with isActions is false to avoid adding it everywhere